tests/unit/plugins/modules/test_cpanm.py

```diff
@@ -36,8 +37,8 @@ def patch_cpanm(mocker):
             'id': 'install_dancer_compatibility',
             'run_command.calls': [
                 (
-                    ['perl', '-le', 'use Dancer;'],
-                    {'environ_update': {}, 'check_rc': False},
+                    ['/testbin/cpanm', '-le', 'use Dancer;'],
```

Comment:
So the module used to run `perl`, and now it runs `cpanm`.... is that intentional? Looks like a potentially breaking change to me.

Reply:
Not a breaking change.

The module continues to run perl, but:
1. This perl call is now performed with `CmdRunner` (it was calling `AnsibleModule.run_command()` previously)
2. `CmdRunner` runs `get_bin_path()` for every command
3. `get_bin_path()` is mocked to `/testbin/cpanm`

So the module itself continues to do the same exact things, but the test had to be adapted.